charts/fybrik/files/taxonomy/data_catalog_response.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Catalog Response",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "Asset": {
+      "$ref": "#/definitions/AssetSpec"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "name",
+    "Asset"
+  ],
+  "definitions": {
+    "AssetSpec": {
+      "type": "object",
+      "properties": {
+        "secretRef": {
+          "$ref": "#/definitions/SecretRef"
+        },
+        "assetMetadata": {
+          "$ref": "#/definitions/AssetMetadata"
+        },
+        "assetDetails": {
+          "$ref": "#/definitions/AssetDetails"
+        }
+      },
+      "required": [
+        "secretRef",
+        "assetMetadata",
+        "assetDetails"
+      ]
+    },
+    "SecretRef": {
+      "$ref": "taxonomy.json#/definitions/Authentication"
+    },
+    "AssetDetails": {
+      "type": "object",
+      "description": "Asset details",
+      "properties": {
+        "connection": {
+          "$ref": "taxonomy.json#/definitions/Connection"
+        },
+        "dataFormat": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "connection"
+      ]
+    },
+    "AssetMetadata": {
+      "type": "object",
+      "properties": {
+        "owner": {
+          "type": "string"
+        },
+        "geography": {
+          "type": "string"
```

Comment:
geography should be required

Reply:
Not sure that is correct for all cases, the control plane should be able to work even if it is not mentioned.

Reply:
The control plane makes many decisions based on the geography of the dataset: from governance actions to config policies and deployment optimization. If this field is not required, people who write governance policies and admin policies need to be aware of this and write the policies appropriately, otherwise predicates will fail on an empty geography value.

Reply:
> If this field is not required, people who write governance policies and admin policies need to be aware of this and write the policies appropriately, otherwise predicates will fail on an empty geography value.

This is still reasonable. I can only guess that a geography field is hardly supported in existing catalogs. It's also not required by all use cases. In my opinion we should be able to support this case as if geography is specified as "unknown" and the different personas need to be aware of it. However for this to work I would actually accept your suggestion to make this required so there will be control through taxonomy of whether "unknown" is a valid value.

Reply:
@rohithdv currently there are no required fields here. The above discussion is about `geography` but we should think if `metadata` should be required too.